pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1800,6 +1779,20 @@ func formatDomainIOThreadPin(vmi *v1.VirtualMachineInstance, domain *Domain, c *
 	return nil
 }
 
+func indexNetworksByName(networks []v1.Network) (map[string]*v1.Network, error) {
+	netsByName := map[string]*v1.Network{}
+	for _, network := range networks {
+		switch {
+		case network.Pod != nil && network.Multus != nil:
```

Comment:
Since you moved the code to a method and called it `indexNetworksByName`, I find it strange to have this validation here. I would introduce another method for this validation or change the method name in some way that will reflect it also has this validation.

Reply:
Good point, this indeed does 2 things. I will create another function that does the validation.

Reply:
Done